src/shared/config.ts

```diff
@@ -92,6 +130,34 @@ const ConvictConfig = Convict<ServiceConfig>({
       format: 'Boolean',
       default: true
     }
+  },
+  // TODO: Add documentation
```

Comment:
TODO

Reply:
I need your help to figure out what the config params mean